src/osw/model/entity.py

```diff
@@ -100,6 +102,28 @@ class Entity(OswBaseModel):
     attachments: Optional[List[str]] = Field(None, title="File attachments")
     meta: Optional[Meta] = None
 
+    def __init__(self, **data):
```

Comment:
Will this code be regenerated by the code generator? I would suggest to move it to OswBaseModel without throwing any Exception (check if label is set => if yes, set name)

Reply:
Recomendation implemented in [commit 0bb9ec95](https://github.com/OpenSemanticLab/osw-python/pull/92/commits/0bb9ec9533864d4c4693087062e7bfb8ebec1ae4)